src/curated-corpus/components/ScheduleSummaryConnector/ScheduleSummaryConnector.tsx

```diff
@@ -112,12 +102,12 @@ export const ScheduleSummaryConnector: React.FC<
       {!data && <HandleApiResponse loading={loading} error={error} />}
       {data &&
         (data.getScheduledCorpusItems.length < 1 ||
-          data.getScheduledCorpusItems[0]?.items.length < 1) && (
+          scheduledItems.length < 1) && (
           <Typography className={classes.heading} variant="h4">
             No stories have been scheduled for this date yet.
           </Typography>
         )}
-      {data && data.getScheduledCorpusItems[0]?.items.length > 0 && (
+      {data && scheduledItems.length > 0 && (
         <>
```

Comment:
Managed to get a couple of far-too-long variable names out of the markup with this refactor.